src/pages/workflows/components/WorkflowRightBoxSection.ts

```diff
@@ -0,0 +1,18 @@
+import { h } from 'react-hyperscript-helpers';
+import { RightBoxSection, RightBoxSectionProps } from 'src/components/RightBoxSection';
+
+export const WorkflowRightBoxSection = (props: RightBoxSectionProps) => {
```

Comment:
Making this file TSX and removing hyperscript helpers, you can simplify this to the code below.

But I would recommend that you simply call `RightBoxSection` explicitly in your usage, as the wrapper component isn't adding anything.

```
export const WorkflowRightBoxSection = (props: RightBoxSectionProps) => {
  return (
    <RightBoxSection {...props}>
      {props.children}
    </RightBoxSection>
  );
};